### ISSUE TITLE
[Media design template] MicroProfile Compatible Implementation Announcement

### ISSUE THREAD

**Issue by aeiras** [microprofile/microprofile-marketing#455]
- Not to be modified 
- example of usage:  [MP tweet ](https://twitter.com/MicroProfileIO/status/1370432229051047938?s=20)
- description as stated on the 7 MP Project listed in this space.

**Comment by rstjames** [comment]
@aeiras I do not understand what you wrote.

**Comment by aeiras** [comment]
@rstjames, 

How about creating a design that is static (no need to change anything in the design, we need something scalable) that could be use with this twitter Annoucements? No design work after its creation. We could create a few versions and just use them randomly as well. :) 

<img width="587" alt="image" src="https://user-images.githubusercontent.com/10731224/111681743-d7549200-87e0-11eb-92be-dad873601d9e.png">

**Comment by rstjames** [comment]
won't we have a compatible logo/ badge? I rather wait to use that on the design before we make this and need to change it later. 
Rough idea.... 
![Screen Shot 2021-03-22 at 1 26 22 PM](https://user-images.githubusercontent.com/12132146/111989616-4c420780-8b12-11eb-8641-c56a133b70f9.png)

**Comment by aeiras** [comment]
We don't know yet if MP will choose to have and use a compatible logo, Ryan. 

I like the design. 
My feedback will be to maybe make it light it and add MP code from the MP Starter here.  Do keep the puzzle in it, a very nice add-on. I think when we think compatible, it means lots of testing, lots of code, and lots of maintenance with people behind it all. 

Cheers,

**Comment by rstjames** [comment]
@aeiras what's the issue with choosing a compatible logo? let's just push it and start to use something. A solid design is already done.

**Comment by rstjames** [comment]
let's hold off on this design. It's not so common to get a new compatible implementation, is it? The reason I say this is that the compatible logo conversation came up again on last night's community call and if we do push that forward I would wait on this ticket.

**Comment by aeiras** [comment]
@rstjames transfered to MPWG repo, lets continue the discussion in that space with follow up updates. Thanks!
